ci-operator/templates/openshift/openshift-azure/build-azure-image.yaml

```diff
@@ -141,6 +141,7 @@ objects:
           -e openshift_azure_container=images \
           -e openshift_azure_vm_ssh_public_key="~/.ssh/id_rsa.pub" \
           -e openshift_azure_vm_size=$AZURE_VM_SIZE \
+          -e @/usr/share/ansible/openshift-ansible/inventory/dynamic/injected/certs.yaml
```

Comment:
@kwoodson before I test this I'd like to know whether ansible is going to log the content of this somewhere in stdout.

Reply:
I don't believe that it will.  The specific yum task has `no_log: True`:

https://github.com/openshift/openshift-ansible/blob/master/playbooks/azure/openshift-cluster/tasks/yum_certs.yml#L12